cli/pcluster/easyconfig.py

```diff
@@ -305,6 +296,73 @@ def _is_config_valid(args, config):
         return False
 
 
+def get_parameter(config, section, parameter_name, default_value):
+    """
+    Prompt the user to ask question without validation
+    :param config the configuration parser
+    :param section the name of the section
+    :param parameter_name: the name of the parameter
+    :param default_value: the default string to ask the user
+    :return:
+    """
+    return config.get(section, parameter_name) if config.has_option(section, parameter_name) else default_value
+
+
+def general_scheduler_handler(config, cluster_template):
```

Comment:
I'd create a function `_init_scaling_params(..., scheduler)` that remove or add the correct parameters according to the scheduler